saas/api/metrics.py

```diff
@@ -22,13 +22,17 @@
 # OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from django.core.serializers.json import DjangoJSONEncoder
 from django.db.models import Q
+from django.utils.datastructures import SortedDict
 from django.utils.dateparse import parse_datetime
+import json
```

Comment:
We try to follow the convention to group imports in block:
- all imports from standard library
- all imports from 3rd party packages
- all imports from project itself (here we haven't very consistent btw using absolute imports and relative (i.e. from . import X)).